src/clib/lib/job_queue/torque_driver.cpp

```diff
@@ -317,6 +340,15 @@ stringlist_type *torque_driver_alloc_cmd(torque_driver_type *driver,
         free(resource_string);
     }
 
+    if (driver->max_runtime != 0) {
+        stringlist_append_copy(argv, "-l");
+        char *walltime_string;
+        walltime_string =
+            util_alloc_sprintf("walltime=%d", driver->max_runtime);
```

Comment:
I guess that `walltime` was tested - is it in seconds?

Reply:
little handy habit i learned from prometheus was to add units to your measurement variable names, but i'm not sure if it's applicable / practical here

Reply:
yes, it is in seconds, goos idea to add units @valentin-krasontovitsch, thank you! I think I will refactor